<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>kMyMoneyOnlineQuoteConfigDecl</class>
  <widget class="QWidget" name="kMyMoneyOnlineQuoteConfigDecl">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>594</width>
        <height>487</height>
      </rect>
    </property>
    <property name="minimumSize">
      <size>
        <width>0</width>
        <height>0</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>Online Quotes</string>
    </property>
    <layout class="QVBoxLayout">
      <item>
        <widget class="K3ListView" name="m_quoteSourceList">
          <property name="sizePolicy">
            <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>3</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="allColumnsShowFocus">
            <bool>true</bool>
          </property>
          <property name="defaultRenameAction">
            <enum>Accept</enum>
          </property>
          <property name="itemsRenameable">
            <bool>true</bool>
          </property>
          <column>
            <property name="text">
              <string comment="@title header of the qoute source name column">Name</string>
            </property>
            <property name="clickable">
              <bool>true</bool>
            </property>
            <property name="resizable">
              <bool>true</bool>
            </property>
          </column>
        </widget>
      </item>
      <item>
        <widget class="QGroupBox" name="groupParsing">
          <property name="title">
            <string>Details</string>
          </property>
          <layout class="QVBoxLayout">
            <item>
              <layout class="QGridLayout">
                <item row="3" column="0">
                  <widget class="QLabel" name="textLabel4">
                    <property name="text">
                      <string>Date</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item row="1" column="0">
                  <widget class="QLabel" name="textLabel2">
                    <property name="text">
                      <string>Symbol</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item row="1" column="1">
                  <widget class="KLineEdit" name="m_editSymbol">
                    <property name="toolTip" stdset="0">
                      <string>Regular Expression to extract the symbol from the downloaded data</string>
                    </property>
                  </widget>
                </item>
                <item row="2" column="1">
                  <widget class="KLineEdit" name="m_editPrice">
                    <property name="toolTip" stdset="0">
                      <string>Regular Expression to extract the price from the downloaded data</string>
                    </property>
                  </widget>
                </item>
                <item row="4" column="1">
                  <widget class="KLineEdit" name="m_editDateFormat">
                    <property name="toolTip" stdset="0">
                      <string>Regular Expression to extract the date from the downloaded data</string>
                    </property>
                  </widget>
                </item>
                <item row="0" column="1">
                  <widget class="KLineEdit" name="m_editURL">
                    <property name="toolTip" stdset="0">
                      <string>URL to be used to download the quote</string>
                    </property>
                    <property name="whatsThis" stdset="0">
                      <string>Enter the URL from which stock quotes will be fetched.  &lt;b&gt;%1&lt;/b&gt; will be replaced with the symbol for the security being quoted. For currency conversions, &lt;b&gt;%2&lt;/b&gt; will be replaced with the currency to be quoted and &lt;b&gt;%1&lt;/b&gt; with the currency the quote is based on.</string>
                    </property>
                  </widget>
                </item>
                <item row="2" column="0">
                  <widget class="QLabel" name="textLabel3">
                    <property name="text">
                      <string>Price</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item row="3" column="1">
                  <widget class="KLineEdit" name="m_editDate">
                    <property name="toolTip" stdset="0">
                      <string>Regular Expression to extract the date from the downloaded data</string>
                    </property>
                  </widget>
                </item>
                <item row="4" column="0">
                  <widget class="QLabel" name="textLabel4_2">
                    <property name="text">
                      <string>Date Format</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item row="0" column="0">
                  <widget class="QLabel" name="textLabel1">
                    <property name="text">
                      <string>URL</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
            <item>
              <widget class="QLabel" name="textLabel5">
                <property name="text">
                  <string>&lt;i&gt;Enter regular expressions which can be used to parse the data returned from the URL entered above.  The symbol, price, and date must be found in the quote data to be usable. You may also try the KMyMoney user's mailinglist at &lt;a href=&quot;mailto:kmymoney2-user@lists.sourceforge.net&quot;&gt;kmymoney2-user@lists.sourceforge.net&lt;/a&gt; to find what settings work for other users in your country.&lt;/i&gt;</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <item>
            <widget class="KPushButton" name="m_newButton">
              <property name="text">
                <string comment="@action new quote source">New</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="KPushButton" name="m_deleteButton">
              <property name="text">
                <string>Delete</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="spacer6">
              <property name="sizeHint">
                <size>
                  <width>240</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="KPushButton" name="m_updateButton">
              <property name="text">
                <string comment="@action update quotes from online source">Update</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
    <customwidgets>
    <customwidget>
      <class>K3ListView</class>
      <extends>Q3ListView</extends>
      <header location="global">k3listview.h</header>
    </customwidget>
  </customwidgets>
</ui>
